### ISSUE TITLE
basic_filebuf::underflow error reading the file: iostream error

### ISSUE THREAD

**Issue by gravisxv** [xmrig/xmrig#2635]
Hello i am trying to mine ravencoin on 290x card. All the settings are ok, i simply start mining correctly until this message appear:
basic_filebuf::underflow error reading the file: iostream error

I have not found any workaround. The problem is not related to the videocard or Os as nanominer is working ok.

Currently im using Manjaro 20.1 and 20.45 amdgpu drivers.

**Comment by Spudz76** [comment]
I wouldn't use newer than 20.30-1109583 on anything Hawaii or older.  But not likely the problem.

Never, ever, ever seen that error before, are you sure it's from xmrig?

**Comment by gravisxv** [comment]
Yep, but i just fixed it. I build the source directly in the menu. It seems the binaries in the download section are not compatible with Manjaro or with Kawpow algo.

**Comment by Spudz76** [comment]
Ah, yes, those are for Ubuntu/Debian mostly.  Who knows what might be different... probably glibc where all that filestream stuff lives...
